WordPress/gutenberg
# ColorGradientControl: Fix awkward padding in popover

Part of #43014

## What?

Fixes the awkward padding when the ColorGradientControl's tab switcher appears in a popover (https://github.com/WordPress/gutenberg/pull/41900#issuecomment-1193795265).

## Why?

There is actually a [further redesign](https://github.com/WordPress/gutenberg/pull/41900#issuecomment-1205124434) in the works, but that is going to take a little more time to achieve. So here is an easy, incremental step to make it less weird in the meantime.

## How?

- Remove popover padding so the tab switcher goes edge-to-edge.
- Always add padding to the panels when they are either within a TabPanel, or within a popover.

## Testing Instructions

1. `npm run dev`
2. In the post editor, add a Code block. In the Inspector, see the color popovers for Text and Background.
3. In the Global Styles sidebar, check the Colors settings.

## Screenshots or screencast 

### Before

<img width="360" alt="The tab panel switcher has too much padding" src="https://user-images.githubusercontent.com/846565/180743220-779c6c0f-bbae-4f3b-b95c-a0852bfaeb87.png">

### After

https://user-images.githubusercontent.com/555336/183015280-a0431083-b475-455c-9cfb-cf7e324f82a3.mp4
